setup.py

```diff
@@ -5,8 +5,11 @@
 from os import path
 
 description_file = path.join(path.abspath(path.dirname(__file__)), "DESCRIPTION.md")
-with open(description_file, encoding="utf-8") as f:
-    long_description = f.read()
+if os.path.exists(description_file):
```

Comment:
While this check should also work around the problem, I don't like that the description may be silently left empty. It's better to fail with an error, and fix the root cause.

Reply:
I strictly agree with you, but I've feel encouraged after seeing the setup of the package [grayskull](https://github.com/conda-incubator/grayskull/blob/main/setup.py). I'll update the PR to remove it, may just the change to the package-data line makes the thing.